splice_machine/src/test/java/com/splicemachine/client/SpliceClientTest.java

```diff
@@ -1,23 +1,21 @@
 package com.splicemachine.client;
 
+import com.splicemachine.db.client.am.SqlException;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
 public class SpliceClientTest {
 
     @Test
-    public void testMaskJDBCPassword() {
+    public void testParseJDBCPassword() throws SqlException {
+        String password = "Abd98*@80EFg";
         String raw = "No suitable driver found for jdbc:splice://dwbdtest1r1w3.wellpoint" +
```

Comment:
Please replace the host name. Use generic names instead.

Reply:
@wtang16 Fixed